controllers/velero.go

```diff
@@ -232,6 +87,9 @@ func (r *DPAReconciler) ReconcileVeleroDeployment(log logr.Logger) (bool, error)
 		// Setting controller owner reference on the velero deployment
 		return controllerutil.SetControllerReference(&dpa, veleroDeployment, r.Scheme)
 	})
+	// if op != controllerutil.OperationResultNone {  // for debugging purposes
+	// 	fmt.Println(cmp.Diff(orig, veleroDeployment))
+	// }
```

Comment:
Should the above be removed?

Reply:
I guess.. when would it ever make sense to leave these in? if its not commented out but rather a debug flag?

Reply:
If there's a useful debug flag to hide this behind, that would be fine too. But it's generally better to avoid having commented-out code in the repo.